opengemini_client/client_impl.py

```diff
@@ -129,11 +129,16 @@ def query(self, query: Query) -> QueryResult:
             results = []
 
             for result in json_data.get('results', []):
+<<<<<<< HEAD
```

Comment:
The conflict has not been resolved and should not be submitted

Reply:
Unwarranted mistakes